### PR TITLE
chore: update pub script

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   -        |
| 🇨🇳 Chinese |      -     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
     "pre-publish": "npm run test-all -- --skip-build",
     "prettier": "prettier -c --write **/*",
     "pretty-quick": "pretty-quick",
-    "pub": "npm run version && npm run collect-token-statistic && antd-tools run pub-experimental",
+    "pub": "npm run version && npm run collect-token-statistic && antd-tools run pub-alpha",
```
Comment (afc163) [inline package.json:80]: 应该自动识别版本号，这里始终是 pub 就行。

Comment (MadCcc) [reply package.json:80]: 我想是搞个问询了，这个区别主要是 dist-tag 不同

Comment (MadCcc) [reply package.json:80]: 把 antd-tools 的 gulp task 翻新下

**Comment by MadCcc** [comment]
发现有个 config 读取 tag 的逻辑

**Review by li-jia-nan — APPROVED** [review]

File: .antd-tools.config.js [hunk 2]
```diff
@@ -29,4 +29,5 @@ module.exports = {
     finalize: finalizeDist,
   },
   bail: true,
+  tag: 'next',
```
[the remark this answers is not in the document]
Comment (MadCcc) [reply .antd-tools.config.js:32]: 和 version 相关的，分支关联不大，next 分支也可能发的是 experimental 版本

**Pull request merged**